inst/tests/tests.Rraw

```diff
@@ -12791,6 +12846,188 @@ if (.Platform$OS.type=="windows") {
+test(1967.36, data.table(1:5, key = 5L), error = 'must be character')
+
+x = data.table(a = 1:5)
+test(1967.37, x[mult = 'none'], error = 'mult argument can only be')
+test(1967.38, x[roll = c(3, 4)], error = 'roll must be a single')
+test(1967.39, x[roll = NA], error = 'roll must be a single')
+test(1967.40, x[roll = 'furthest'], error = 'Only valid character value')
+test(1967.41, x[rollends = 1 + 3i], error = 'rollends must be a logical')
+test(1967.42, x[rollends = rep(TRUE, 10L)], error = 'rollends must be length 1 or 2')
+test(1967.43, x[ , ..], error = 'symbol .. is invalid')
+test(1967.44, x[NULL], data.table(NULL))
+test(1967.45, x[ , NULL], NULL)
+test(1967.46, x[ , 'b' := 6:10, with = FALSE],
+     data.table(a = 1:5, b = 6:10), warning = 'with=FALSE ignored')
+test(1967.47, x[ , -1L, with = FALSE], data.table(b = 6:10))
+test(1967.48, x[ , b, .SDcols = 'a'], 6:10,
+     warning = "This j doesn't use .SD")
+test(1967.49, x[ , list(5) := 6], error = 'LHS of := must be a symbol')
+test(1967.50, x[ , 1 + 3i := 6], error = "LHS of := isn't column names")
+test(1967.51, x[ , .(5L), by = .EACHI, mult = 'all'],
+     error = 'logical error. i is not data.table')
+#### to be removed following #2061
+test(1967.9999, x[1:2, by = a], error = "'by' or 'keyby' is supplied but not j")
+test(1967.52, x[1+3i], error = 'i has not evaluated to logical')
+
+test(1967.53, as.matrix(x, rownames = 2:3),
+     error = 'length(rownames)==2 but')
+test(1967.54, as.matrix(x[0L]),
+     structure(logical(0), .Dim = c(0L, 2L), .Dimnames = list(NULL, c("a", "b"))))
+
+test(1967.55, subset(x, 5L), error = "'subset' must evaluate to logical")
+
+x = as.list(x)
+test(1967.56, setnames(x), error = 'x is not a data.table or data.frame')
+setDT(x)
+names(x) = NULL
+test(1967.57, setnames(x), error = 'x is length 2 but its names are length 0')
+names(x) = c('a', 'b')
+test(1967.58, setnames(x, new = 'c'), error = "When 'new' is provided")
+test(1967.59, setnames(x, 1:2, c(8L, 9L)), error = "'new' is not a character")
+test(1967.60, setnames(x, -1:1, c('hey', 'you')), error = "both +ve and -ve")
+test(1967.61, setnames(x, 1+3i, 'cplx'), error = "'old' is type complex")
+test(1967.62, setnames(x, 1, c('d', 'e')), error = "'old' is length 1 but 'new'")
+
+test(1967.63, setcolorder(x, c(1, 1)), error = 'contains duplicates')
+test(1967.64, setcolorder(x, 1+3i), error = 'not a character or numeric')
+test(1967.65, setcolorder(x, 300), error = 'Column numbers in neworder out of bounds')
+
+test(1967.66, rbindlist(list(x), idcol = FALSE),
+     rbindlist(list(x)))
+test(1967.67, rbindlist(list(x), idcol = 1+3i), error = 'idcol must be a logical')
+
+### potentially altering user environment so use a strange name
+suppressWarnings(rm(`___data.table_internal_test_1967.68___`))
+test(1967.68, setDT(`___data.table_internal_test_1967.68___`), error = 'Cannot find symbol')
+
+### [.data.table verbosity & non-equi-join tests
+opt = options('datatable.optimize' = 2)
```

Comment:
@mattdowle @jangorecki just realized I forgot to reset this option in this PR, not sure when I'll get time in case one of you sees this 👍

Reply:
No worries @MichaelChirico.  Done:  https://github.com/Rdatatable/data.table/commit/3acfd80b3a1ee914e86392177b9f1747fc647201